dodo.py

```diff
@@ -10,9 +12,18 @@
 
 DOIT_CONFIG = {"default_tasks": ["build"]}
 
+_ARCH_PLATFORM_MAPPING = {
+    "aarch64": "linux/arm64",
```

Comment:
Should this be in this PR? Or does it belong to #372?

Reply:
This has nothing to do with #372, but running with `--platform arm64` will fail since #372 has not yet applied. 
As you can see from the initial code, it tries to support aarch64 by mapping `aarch64` to `linux/arm64`, it is still valid if run from an amd64 machine and the platform will get from docker info and set to be amd64.